dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/models/SubSnpCoreFields.java

```diff
@@ -374,9 +374,9 @@ private String calculateReverseComplement(String alleleInReverseStrand) {
     public Region getVariantCoordinates() {
         Region variantRegion;
         if (isValidRegion(chromosomeRegion)) {
-            variantRegion = chromosomeRegion;
+            variantRegion = new Region(chromosomeRegion.getChromosome(), chromosomeRegion.getStart(), chromosomeRegion.getEnd());
```

Comment:
Did we do anything this time around to create new object as opposed to the reference?